modules/administration/pages/ssl-certs-hsts.adoc

```diff
@@ -0,0 +1,12 @@
+[[ssl-certs-hsts]]
+= HTTP Strict Transport Security
+
+For additional security with regard to certificates and better protection against specific MITM attacks, it is possible to enable [HTTP Strict-Transport-Security](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security).
+This is done by uncommenting the following line in the Apache Webserver configuration on the {productname} server as well as {productname} proxies if used:
+
+----
+# Enable HSTS
+Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"
+----
+
+Be aware that once HSTS is enabled, browsers will no longer allow users to make exceptions when trusting invalid or self-signed certificates, so in order to be able to access {productname}, the respective CA certificate will need to be imported to browsers of all the users.
```

Comment:
browsers will no longer allow users to make exceptions when trusting invalid or self-signed certificates. To ensure access to the {productname} server, the respective CA certificate will need to be imported to browsers of all users.